### ISSUE TITLE
[QCR][Release ring]Release day pipeline should push private transport packages to public feeds on release day

### ISSUE THREAD

**Issue by mmitche** [dotnet/arcade#5484]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

For an internal build, release day stage of the pipeline, in concert with the push to nuget.org, should take internal transport packages and push them to public feeds so that branding update PRs can complete in public.

It's is not necessary to know whether a build is public or private. Just look for packages that exist on internal transport feeds and push them to the corresponding public feeds. Here is a powershell snippet sample which will do this:

```
builds = Get-Content \\vsufile\patches\sign\NET\CORE_BUILDS\3.1.X\3.1.4\3.1.300-servicing-015161\manifest.json | ConvertFrom-Json;

foreach ($asset in $builds.builds.assets) {
  if ($asset.Source.StartsWith('https://pkgs.dev.azure.com/dnceng/_apis/packaging/feeds/dotnet3.1-internal-transport')) {
    D:\tools\nuget.exe push -Source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" -ApiKey AzureDevOps $asset.targets[0] -SkipDuplicate
  } 
}
```

Feed: mapping:
- dotnet3.1-internal-transport -> dotnet3.1-transport
- dotnet5-internal-transport -> dotnet5-transport

**Comment by jcagme** [comment]
This should happen in the process of publishing/releasing and not staging right?

**Comment by mmitche** [comment]
> This should happen in the process of publishing/releasing and not staging right?

Affirmative.

**Comment by adiaaida** [comment]
Do the symbols also need to be moved? So:

dotnet5-internal-symbols -> dotnet5-symbols

**Comment by adiaaida** [comment]
Do we care about nonshipping assets?

**Comment by mmitche** [comment]
> Do we care about nonshipping assets?

Yeah. The transport packages are the non-shipping assets (just another name for them, really)

**Comment by adiaaida** [comment]
Gotcha

**Comment by mmitche** [comment]
/cc @rbhanda @silvioguiso In case there is some overlap to what they are doing.

@adiaaida are you adding this for 3.1 too?

**Comment by adiaaida** [comment]
Yes. I'm trying to make this as general as possible.

**Comment by adiaaida** [comment]
Reopening this issue and adding to V2. We've got the yaml ready, it just needs the publishing pipeline to exist so that it can be added.

**Comment by wtgodbe** [comment]
CC @Pilchie

**Comment by adiaaida** [comment]
https://dev.azure.com/dnceng/internal/_git/dotnet-release/pullrequest/16705
